added a python script to validate the sonar token before it is generated

RobeDevOps opened the pull request:
* Added new python script to validate the sonar token generation
* If token exists then the process of generation is skipped.

This issue is fixed based on 
https://github.com/Accenture/adop-docker-compose/issues/285

RobeDevOps commented:
Please let me know who can be the reviewer and if anything needs to be fixed or improved.

dsingh07 commented:
Hi @RobeDevOps Thank you for the PR! Looks good to me!

Just wondering if it's possible to extend the existing bash script to deal with the case where a token already exists rather than needing an extra python script to do the checking? I think it will make it easier to debug and decipher if all the logic for the sonar auth token generation is encapsulated within one script. What do you think?

/CC @RobertNorthard

dsingh07 commented:
@RobeDevOps Yes I think, let's proceed with that approach, this will keep it cleaner and make it easier to debug, thanks!

RobeDevOps commented:
@dsingh07 I removed the python script and added the python code as simple extension into the bash script file.

- If the list is empty return empty token so, token is gonna be generated
- Looping the token arrays and it returns the token ( if exists)
- Finally returns empty value if not token found.

Please let me know if that is ok. I think the python script is simple and adding jq to the Dockerfile is going to increase the image size and also a new tool and dependencies.

dsingh07 commented:
Thanks a lot @RobeDevOps that sounds good to me, I'll try and test it as soon as possible and hopefully merge :)

RobertNorthard commented:
Thanks @RobeDevOps - I like the approach

RobeDevOps commented:
I squashed the commits in one. 👍

dsingh07 commented:
@RobeDevOps New 0.3.5 release is created: https://hub.docker.com/r/accenture/adop-jenkins/builds/

Please can you create an accompanying to adop-docker-compose to update the image tag?
